services/search.py

```diff
@@ -16,18 +16,23 @@ def get(self):
 
         s = urllib.unquote_plus(s)
 
+        query = '"{0}"'.format(s.replace('"', ''))
```

Comment:
I was finding with tags using this query. I had to modify it because it was crashing typing commas.

Reply:
Ok. I have a feeling the tag bug was introduced in this file so I'll debug.